bazel/repositories.bzl

```diff
@@ -334,16 +333,6 @@ def _boringssl_fips():
         build_file = "@envoy//bazel/external:boringssl_fips.BUILD",
     )
 
-def _com_github_bombela_backward():
-    _repository_impl(
-        name = "com_github_bombela_backward",
-        build_file = "@envoy//bazel/external:backward.BUILD",
```

Comment:
I just noticed that https://github.com/envoyproxy/envoy/blob/master/bazel/external/backward.BUILD is still in the codebase. This can be removed now I believe @dnoe